Add google season of docs 2023 proposal (#120)
jupyter/accessibility: docs/funding/google-season-of-docs-2023.md

# Proposal Title - organization name

<!---
Your proposal title should be short and specific. “Update ORGNAME Contributor Guide” is a good proposal title. “Documentation Improvements” is too vague; “Update Sections 5,7,23,99 of Contributor Guide, Create FAQ, and Create Style Guide” is too long.
-->

> Create foundational accessibility documentation for JupyterLab

Reviewer: In the spirit of clarity I suggest:

```suggestion
Create foundational user-focused accessibility documentation for JupyterLab
```

# About your organization

<!---
In this section, tell us about your organization or project in a few short paragraphs. What problem does your project solve? Who are your users and contributors? How long has your organization or project been in existence? Give some context to help us understand why funding your proposal would create a positive impact in open source and the world.
-->

JupyterLab is an extensible environment for interactive and reproducible computing based on the Jupyter Notebook. It allows scientists, researchers, engineers, journalists, students, and many other users to configure and arrange workflows combining code development, visualization and text inside a single web-based interface. Characterized by its powerful execution engine, Jupyter supports over 40 programming languages, including Python, Julia, R and Scala.

> JupyterLab is a user interface offering notebook manipulation, text editor, file browser, and terminal, among other development tools. It has a seamless integration with the most common scientific packages and it is included in multiple downstream applications as a dependency. Moreover, JupyterLab is flexible, powerful and extensible by third-party and official extensions distributed via PyPi, conda and other package managers.

Reviewer: ```suggestion
JupyterLab is a user interface comprising notebook manipulation, text editor, file browser, and terminal, among other development tools. It has a seamless integration with the most common scientific packages and is included in multiple downstream applications as a dependency. Moreover, JupyterLab is flexible, robust, and extensible by third-party and official extensions distributed via PyPi, conda, and other package managers.
```

Since its first release in 2018, JupyterLab has grown as an open source project, offering more features than just the Notebook interface. It is used by many applications and it has served as a reference for multiple web-based interfaces like Google Codespaces, Google Collab and Coursera, among others.

One part of that story of growth, adoption, and inclusivity that was overlooked for too long was the inclusion of disabled users and the implementation of accessibility best practices. That’s why during the past years, the Jupyter Accessibility team has been leading efforts to make the JupyterLab interface more accessible and usable by disabled users. There’s still much to do, but it’s vitally important—not just for JupyterLab but for the wider ecosystem of data science and open source—that this work continues.

# About your project

## Your project’s problem

<!---
Tell us about the problem your project will help solve. Why is it important to your organization or project to solve this problem?
-->

> The JupyterLab accessibility team has been concentrating its efforts on accessibility remediation through work on interface audits, testing events, improvements to UI/UX and automated testing. Due to the limited number of resources and the prioritization of such remediation work, documenting accessibility features has been a secondary priority. As more accessibility-focused improvements are added, it is becoming increasingly necessary that we provide a dedicated focus on adding user accessibility documentation for JupyterLab.

Reviewer: ```suggestion
The JupyterLab accessibility team has been concentrating its efforts on accessibility remediation through work on interface audits, testing events, improvements to UI/UX and automated testing. Due to the limited resources and the prioritization of such remediation work, documenting accessibility features has been a secondary priority. As more accessibility-focused improvements are added, it is becoming increasingly necessary that we provide a dedicated focus on adding user accessibility documentation for JupyterLab.
```

> Through this project, we aim to create documentation primarily targeted at anyone with a temporary, situational, or permanent disability that wants or needs to use JupyterLab. We want to provide the JupyterLab community with a comprehensive guide about the accessibility features present in the application. For example, developing documentation that answers the questions “What can and can’t I do in JupyterLab with accessibility accommodations?” and “How can I adjust JupyterLab to work better for me with assistive technology?”

Reviewer: ```suggestion
Through this project, we aim to create documentation primarily targeted at anyone with a temporary, situational, or permanent disability that wants or needs to use JupyterLab. We want to provide the JupyterLab community with a comprehensive guide about the accessibility features present in the application. For example, developing documentation that answers questions such as “What can and can’t I do in JupyterLab with accessibility accommodations?” and “How can I adjust JupyterLab to work better for me with assistive technology?”
```

## Your project’s scope

<!---
Tell us about what documentation your organization will create, update, or improve. If some work is deliberately not being done, include that information as well. Include a time estimate, and whether you have already identified organization volunteers and a technical writer to work with your project.
-->

This project includes:

- Creation of an accessibility section in JupyterLab documentation containing default accessibility settings and customizations inspired by https://code.visualstudio.com/docs/editor/accessibility
- Create user-focused documentation that summarizes the state of JupyterLab areas that have been audited for accessibility.  This includes an approachable report on keyboard navigation and 400% zoom audits results.

As a stretch goal, we would like to include:

- An authoring guide focused on best practices to generate accessible notebooks.

Work that is out of the scope of this project includes:

- Creation of developer-focused documentation
- Tutorials and workshops to present in conferences
- Live streams and social media engagement

> The developer that will be mentoring this project has previously mentored successful GSoD in another project last year and has been involved in the accessibility remediation work for JupyterLab. We don’t have a technical writer for this project, but we intend to start the  application and selection process once we receive the acceptance by Google Season of Docs.

Reviewer: Do we need to name the mentor?

Author: Last year we didn't named anyone in the proposal.

We estimate that this project will be completed by one technical writer working full-time in a span of six months.

## Measuring your project’s success

<!---
How will you know that your new documentation has helped solve your problem? What metrics will you use, and how will you track them?
-->

> Even though JupyterLab users vary in their experience, knowledge and confidence of the application, there’s an overall unfamiliarity about the accessibility features available within it. This is mostly a result of the lack of a reliable and official documentation that can guide users through different situations like adjusting theming, keyboard support and assistive technologies support. There have been numerous questions raised about accessibility features in the issue tracker of the project and in general, the community wants to move to a more inclusive access to the information given that JupyterLab is one of the most common tools used in education, research and scientific development.

Reviewer: ```suggestion
Even though JupyterLab users vary in their experience, knowledge and confidence of the application, there’s an overall unfamiliarity about the accessibility features available within it. This is primarily a result of the lack of reliable and official documentation that can guide users through different situations like adjusting theming, keyboard and assistive technologies support. There have been numerous questions raised about accessibility features in the issue tracker of the project, and in general, the community wants to move to more inclusive access to the information given that JupyterLab is one of the most common tools used in education, research and scientific development.
```

> We will track three metrics to measure this documentation project's success. The first is the number of accessibility features documented; the second is the number of pages added regarding the audits for keyboard navigation and 400% zoom.

Reviewer: This mentions three metrics - though I can only see two

1. No features documented
2. Zoom and keyboard pages

Seems this needs adjusting

```suggestion
We will track two metrics to measure this documentation project's success. The first is the number of accessibility features documented; the second is the number of pages added regarding the audits for keyboard navigation and 400% zoom.
```

We expect open source projects to use open source tools whenever possible; if your project absolutely requires funds for proprietary software licenses or support, please include a justification for the amount.
Other possible expenses include:
Design work to create branding, logos, templates, or other design assets for your documentation site
Minimal amounts (<US$200) for project swag (t-shirts or stickers for your participants). If you use the Season of Docs logo, it must be accompanied by your project or organization logo or name. Your swag may not use the name Google.
Minimal stipends for volunteers who take on considerable mentorship or guidance roles in the project (we recommend no more than $500 per volunteer, please)
Downstream donations to other open source projects should be no more than 10% of your budget total.
Include other budget items as needed, along with justification for the amount sought. Expense justifications should highlight how the expenditure will contribute to the success of the project as a whole.
-->

| Budget item | Amount | Running Total | Notes/Justifications |
| :--:    |      :---:   | :--:    |      :---:   |

> | Technical writer that will update, test and publish new accessibility documentation |$10000 |$10000 |The mentors’ time will be provided as in-kind donation by Quansight Labs so the budget will be used in full for the technical writer’s compensation. |

Reviewer: Maybe add a link to Quansight Labs site
